api/src/main/java/org/openmrs/module/referencedemodata/patient/DemoPatientGenerator.java

```diff
@@ -317,9 +343,38 @@ private Visit createDemoVisit(Patient patient, List<VisitType> visitTypes, Locat
 				visit.addEncounter(createDemoLabsEncounter(patient, toDate(lastEncounterTime), admitLocation));
 			}
 			
+			if (randomDoubleBetween(0.0, 1.0) < .75) {
+				Encounter orderEncounter = null;
+				if (randomDoubleBetween(0.0, 1.0) < .5) {
+					orderEncounter = createDemoOrdersEncounter(patient, toDate(lastEncounterTime), location);
+					List<Concept> testConcepts = getConceptService().getConceptsByClass(getConceptService().getConceptClassByName("Test"));
```

Comment:
Shouldn't we be able to cache the `testConcepts` rather than potentially reloading them?

Reply:
Yes @ibacher...